test/protected_resource_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'test_helper'
+
+class TestResource < Minitest::Test
+  SomeError = Class.new(StandardError)
+
+  def setup
+    Semian.destroy(:testing)
+  rescue
+    nil
+  end
+
+  def teardown
+    destroy_resources
+  end
+
+  def test_acquire_without_bulkhead
```

Comment:
The test says it acquires without bulkhead but I don't see an acquire call in the test.